handbook/product/product_documents.md

```diff
@@ -0,0 +1,65 @@
+
+_All PDs are in a [public Google Drive folder](https://drive.google.com/drive/folders/1Wd-Xx2wNbFtSzeJwbZqMOxdbFDUFxlyR)._
+
+## PDs have similar properties to RFCs
+
+They are:
+
+- [Sequentially numbered](../communication/rfcs.md#rfcs-are-sequentially-numbered)
+- [Google Docs](../communication/rfcs.md#rfcs-are-google-docs)
+- [Public](../communication/rfcs.md#rfcs-are-public)
+- [Open to external contributions](../communication/rfcs.md#external-contributors)
+
+## Status
+
+Each PD has a status that is in the title of the PD (e.g. "PD 1 WIP: Title"). The author is responsible for keeping the status updated.
+
+| Status | Description |
+|-------|-------------|
+| WIP | The author is still drafting the PD and it is not ready for review. |
+| REVIEW | The PD is ready to be reviewed. The PD explicitly lists whose approvals are required and a requested timeline for those approvals. |
+| DEFINED | The problem statement defined and agreed upon, and is locked to changes. This is to ensure that anyone involved in solving the problem statement defined within the PD does not need to worry about changing scope. A change to the problem statement likely requires a new PD to be created to define the new problem statement. If the update is minor enough, all interested parties should be notified. |
+| ABANDONED | There are no plans to move forward with solving the problem statement defined in the PD. The particular reason is communicated in the metadata section of the PD. For example, the PD may have failed to get the necessary approvals, it may be been superseded by another PD, priorities may have changed, or we may not have resources to work on this PD in the foreseeable future. |
+| SOLVED | The problem statement defined in the PD has been solved. |
+
+## PD structure
+
+Effective PDs contain the following information. The below might feel like a heavy structure, this is intentional to help ensure all important considerations and context has been written down. Many sub-sections are very short, and some can be omitted if not relevant to the specific PD.
+
+_For convenience, there is a [Google Docs Template](https://docs.google.com/document/d/1MBZxnRlDG69Fyvzpai5rBqxizvX5zVeZiUe6z7VZrjk/edit?usp=sharing) that can be used when creating new PDs. The template format should help clarify the below structure._
+
+- Title that includes the PD number.
+  - The title is inlined in the Google Doc so that it is more visible and will not disappear if exported to a different format.
+- Metadata about the state of the PD. Including but not limited to:
+  - **Editor:** The person responsible for iterating on the content of the RFC.
+  - **Status:** A description of the current state or outcome of the PD.
+  - **Requested approvers:** The list of people that the PD author is requesting a review from and a requested deadline for those reviews.
+  - **Approved by:** A list of people who approve the problem statement defined in the PD.
```

Comment:
Is there an implication that approval means "we should do this now" or just "we should solve this problem"?